package.json

```diff
@@ -20,7 +20,7 @@
     "setup"
   ],
   "dependencies": {
-    "@actions/cache": "^3.0.6",
+    "@actions/cache": "^3.2.4",
```

Comment:
This is the only change I made. All other changes came from running the `npm package` command